report/src/sections/07-future-works.tex

```diff
@@ -7,8 +7,14 @@ \section{Future Works}\label{sec:future-works}
 The extension of the abstract interpretation to support floating point numbers would allow for a more precise analysis of the database queries.
 One of the obstacles to this extension is the fact that floating point numbers are represented in a different ways in different programming languages, which makes it difficult to create a general abstract interpretation for floating point numbers.
 
-Another area that could be further developed is the precision of the semantic in som cases.
-The current set of semantics are very similar to the ones fund in the original paper by Halder and Cortesi~\cite{halder_abstract_2012}, and they are not very precise.
-This could then lead to a more precise analysis of the database area and more accurate results when judging the abstract interpretation of the database.
+Another area that could be further developed is the precision of the soundness proof.
+Since you can keep reducing the area represented by the abstract interpretation, given more time it would be natural to assume that the soundness proof could be made more concise to further narrow down the area represented by the abstract interpretation.
```

Comment:
This is mixing the precision on the analysis with the rigor of the soundness proof. This should be two different points.